plugins/modules/vmware_guest_cross_vc_clone.py

```diff
@@ -307,6 +312,9 @@ def populate_specs(self):
         creds.password = self.destination_vcenter_password
         self.service_locator.credential = creds
 
+        if self.destination_vcenter_thumbprint:
+          self.service_locator.sslThumbprint = self.destination_vcenter_thumbprint
```

Comment:
```suggestion
            self.service_locator.sslThumbprint = self.destination_vcenter_thumbprint
```

Looks like the [linter doesn't like your indentation](https://ansible.softwarefactory-project.io/zuul/build/1724bf3e5192420ea55a6cd71774cdd2).